frontend/package.json

```
@@ -4,7 +4,7 @@
   "description": "NEAR blockchain explorer",
   "scripts": {
     "typecheck": "tsc --noEmit",
-    "test": "npm run typecheck && jest",
+    "test": "npm run typecheck && TZ=UTC jest",
```

We need that for proper time faking. Do you have any ideas where I should put a comment about that?

---

Yeah JSON is not great at giving ways to leave comments for humans 😞 

I don’t have anything to offer here except ugly hack of creating .test-comment key next to it and write the comment inside or have echo command before setting TZ.

Also, instead of relying on shell to support setting env variables, use env:

```suggestion
    "test": "npm run typecheck && env TZ=UTC jest",
```

---

I hope a person who'll be curious why `TZ=UTC` was added will make his way to this PR for explanation.

> Also, instead of relying on shell to support setting env variables, use env

Ok, I didn't have much experience with cross-system env variables